src/hotspot/share/runtime/arguments.cpp

```diff
@@ -93,6 +92,8 @@ bool   Arguments::_enable_preview               = false;
 char*  Arguments::SharedArchivePath             = NULL;
 char*  Arguments::SharedDynamicArchivePath      = NULL;
 
+LegacyGCLogging Arguments::_legacyGCLogging     = { 0 };
```

Comment:
`{ 0, 0}` ? Or it implied the other members are zero-initialized?

Reply:
@dholmes-ora I think compiler will assign 0 to all members of this struct. Since you have mentioned this here, let's explicitly use { 0, 0} to avoid potential problems.